packages/venia-concept/src/index.css

```diff
@@ -88,3 +88,50 @@ button {
     touch-action: manipulation;
     -webkit-appearance: none;
 }
+
+[data-title] {
```

Comment:
Checking with @jimbo but I think we want to avoid styling against any selectors but classes and pseudo-elements, for the simple reason that the classes object can be exported for extension and other selectors can't. Can you add a class to elements with a `data-title` attribute instead?